internal/flags/constants.go

```diff
@@ -18,10 +18,11 @@ package flags
 const (
 	// UserManagement enables user management, i.e. invitations, role assignments, etc.
 	UserManagement Experiment = "user_management"
-	// EvalHistory enables logging of evaluation history in the new tables.
-	EvalHistory Experiment = "eval_history"
 	// DockerHubProvider enables the DockerHub provider.
 	DockerHubProvider Experiment = "dockerhub_provider"
 	// GitLabProvider enables the GitLab provider.
 	GitLabProvider Experiment = "gitlab_provider"
+	// ImprovedEvalDetails enables improved evaluation details
+	// messages.
+	ImprovedEvalDetails Experiment = "improved_eval_details"
 )
```

Comment:
I'd be inclined to make this experiment specific to the particular improvement, rather than trying to cover _all_ eval conditions with one flag.  In particular, it will make retiring the experiment later easier, because the scope will be limited.

Reply:
e.g. 
```suggestion
	// VulnCheckErrorTemplate enables improved evaluation details
	// messages in the vulncheck rule.
	VulnCheckErrorTemplate Experiment = "vulncheck_error_template"
)
```